tests/unit/helpers.js

```diff
@@ -1470,3 +1474,68 @@ export const createExperimentData = ({
 }) => {
   return { [id]: variantId };
 };
+
+const queryByClassName = (container, className) => {
+  return container.querySelectorAll(`.${className}`);
+};
+
+const getByClassName = (container, className) => {
+  const elements = queryByClassName(container, className);
+  if (!elements.length) {
+    return null;
+  }
+  if (elements.length > 1) {
+    throw new Error(`More than one element found for className ${className}`);
+  }
+  return elements[0];
+};
+
+const queryByTagName = (container, tagName) => {
+  return container.getElementsByTagName(tagName);
+};
+
+const getByTagName = (container, tagName) => {
+  const elements = queryByTagName(container, tagName);
+  if (!elements.length) {
+    return null;
+  }
+  if (elements.length > 1) {
+    throw new Error(`More than one element found for tag ${tagName}`);
+  }
+  return elements[0];
+};
+
+export const render = (ui, options = {}) => {
```

Comment:
Some notes:

- Many (in fact most) of our components use the `translate` HOC which ends up injecting `i18n` into the component. This helper function wraps the component under test with `I18nProvider` which does the same thing. 
- A lot of our components are connected to the redux store. Wrapping the component under test with `Provider` ensures that the `store` is available inside the component during the test.
- A lot of our components receive `history` and/or `location` information from the `connected-react-router` component.  Wrapping the component under test with `ConnectedRouter` ensures that theses are available inside the component during the test.

Not all components require all of these. There are many combinations (e.g., a component might need `i18n` and `store`, but not the router info, whereas another might need `store` and the router info). We could complicate this `render` helper by making it smart enough to only wrap with the necessary providers, but that would make it more complex and would also require the tests to configure that. I am taking the approach here of just always wrapping everything with all 3 HOCs. I *think* this is okay, and shouldn't invalidate any tests we write, but I am interested in feedback from the team about whether you think this is a reasonable way to go, or if we should make sure that whatever component we are testing is *only* wrapped with the HOCs that it actually uses in the app.